.pre-commit-config.yaml

```diff
@@ -0,0 +1,26 @@
+repos:
+  - repo: https://github.com/psf/black
+    rev: 24.3.0
+    hooks:
+      - id: black
+        args: ["--target-version", "py38"]
+  - repo: https://github.com/PyCQA/isort
+    rev: 5.13.2
+    hooks:
+      - id: isort
+        args: ["--profile", "black"]
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: v3.1.0
+    hooks:
+      - id: prettier
+        args: [--prose-wrap=preserve, --print-width=90]
+  - repo: https://github.com/pycqa/flake8
+    rev: 7.0.0
+    hooks:
+      - id: flake8
+        args: ["--max-line-length=90", "--extend-ignore=E203,W503"]
```

Comment:
And I think W503 is not needed? https://github.com/psf/black/issues/1660

Reply:
i've moved args to separate file which is accepted by both pre-commit and VSC flake8 extension. 
i was never able to persuade pycharm to use it, yet anyways